dash_auth/plotly_auth.py

```diff
@@ -196,7 +196,7 @@ def create_or_overwrite_oauth_app(app_url, name):
 
     res = api_requests.post('/v2/oauth-apps/update_or_create', **request_data)
 
-    if res.status_code != 404:
+    if res.status_code not in [404, 405]:
```

Comment:
I _think_ that this can just be `if res.status_code != 405`

Reply:
@chriddyp just tested this out, it's working. Aside from the issues with the tests this should be good to go.

Reply:
@chriddyp And now it looks like 3.6 is failing and 2.7 is passing 🙃